### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by Itsjustblack**
Removed the Paystack API key from .env file
Implemented the fixes from the last review session

### CODE REVIEW

File: src/app/api/product/checkout/route.ts
Comment (jideabdqudus) [inline src/app/api/product/checkout/route.ts:1]: I'll suggest creating a types folder within the root, and moving the typescript types in there for each file.

File: src/app/(home)/page.tsx [hunk 1]
```diff
@@ -47,10 +48,10 @@ export default async function Home() {
       <QueryErrorBoundary>
         <Suspense
           fallback={
-            <div className="grid grid-cols-1 md:grid-cols-2 gap-10 lg:grid-cols-3 justify-items-center justify-between animate-pulse mt-16">
-              <div className="bg-gray-700 max-h-72 h-72 w-full rounded-xl"></div>
-              <div className="bg-gray-700 max-h-72 h-72 w-full rounded-xl"></div>
-              <div className="bg-gray-700 max-h-72 h-72 w-full rounded-xl"></div>
+            <div className="grid grid-cols-1 md:grid-cols-2 gap-10 lg:grid-cols-3 justify-items-center justify-between mt-16">
+              <Skeleton className="bg-[#1e1e1e] max-h-72 h-72 w-full rounded-xl" />
```
Comment (jideabdqudus) [inline src/app/(home)/page.tsx:52]: Would it be worth creating a variable for the color hexcodes so it conforms with the pattern of the rest of the codebase.

File: src/components/ModalContainer.tsx [hunk 2]
```diff
@@ -21,13 +21,22 @@ export default function ModalContainer({ modal, className, children }: Props) {
...
+  );
+
   return (
     <Transition show={isOpen} as={Fragment}>
       <Dialog
         as="div"
         className="relative z-50"
         // initialFocus={cancelButtonRef}
-        onClose={closeModal}
+        onClose={() => {}}
```
Comment (jideabdqudus) [inline src/components/ModalContainer.tsx:39]: Not sure what we are going for here or you meant to remove the onClose function

**Review by jideabdqudus — COMMENTED** [review]
Lgtm

**Comment by jideabdqudus** [comment]
Are we making use of refresh token within the app yet? @Itsjustblack @Adebayo-Adesegun

**Review by jideabdqudus — APPROVED** [review]

**Comment by binectics** [comment]
> Are we making use of refresh token within the app yet? @Itsjustblack @Adebayo-Adesegun

Not yet @jideabdqudus, just the initial JWT token for instances where its needed

**Pull request merged**